src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -64,19 +65,38 @@
     :blockquote
     (conj blocks
           [rn/view {:style style/quote}
-           [quo/text literal]])
+           [quo/text literal]]
+          (when edited-at
+            [quo/text (style/edited-style)
+             (str " (" (i18n/label :t/edited) ")")]))
 
     :codeblock
     (conj blocks
           [rn/view {:style (merge style/block (style/code))}
-           [quo/text (subs literal 0 (dec (count literal)))]])
+           [quo/text (subs literal 0 (dec (count literal)))]]
+          (when edited-at
+            [quo/text (style/edited-style)
```

Comment:
properties should be used instead of styles

Reply:
Sorry for missing this, long time bugfixing and using quo 😂
Will fix

Reply:
> properties should be used instead of styles

🎯

Reply:
Done